snapshot/lib/snapshot.rb

```diff
@@ -17,3 +17,46 @@
 require_relative 'snapshot/simulator_launchers/simulator_launcher'
 require_relative 'snapshot/simulator_launchers/simulator_launcher_xcode_8'
 require_relative 'snapshot/module'
+
+require 'fastlane_core'
+
+require 'open3'
+
+module Snapshot
```

Comment:
a `Snapshot` module should be placed in `snapshot/module.rb` relative from here,